src/methods/function.js

```diff
@@ -1122,6 +1128,7 @@ export class FunctionImplementation {
   // If c is undefined, the result is just realm.savedCompletion.
   // Call this only when a join point has been reached.
   incorporateSavedCompletion(realm: Realm, c: void | AbruptCompletion | Value): void | Completion | Value {
+    invariant(!(c instanceof SimpleNormalCompletion), "TODO: This function doesn't accept NormalCompletions");
```

Comment:
The type annotation already precludes c from being a normal completion. There is no need for it and there is no todo here either.